Monika After Story/game/script-apologies.rpy

```diff
@@ -0,0 +1,250 @@
+        9: "the game crashing. I understand it happens sometimes, but don't worry, I'm alright!",
+        10: "the game crashing. It really was scary, but I'm just glad you came back to me and made things better."
+    }
+
+    #We want to make sure that the None possibility is still a thing after apologizing once.
+    if mas_apology_reason is not None:
+        $ apology_reason = mas_apology_reason_db.get(mas_apology_reason,mas_apology_reason_db[0])
+
+    #If there's no reason to apologize
+    if mas_apology_reason is None and persistent._mas_apology_time_db == {}:
+        m 1ekd "Did something happen?"
+        m 2ekc "I see no reason for you to be sorry."
+        m 1dsc "..."
+        m 1eub "Anyway, thank you for the apology."
+        m 1eua "Whatever it is, I know you're doing your best to make things right."
+        m 1hub "That's why I love you, [player]!"
+
+    #She knows what you are apologizing for
+    elif mas_apology_reason:
+        m 1eka "Thank you for apologizing for [apology_reason]"
+        m "I accept your apology, [player]. It means a lot to me."
+
+    #She knows that you've got something else to apologize for, and wants you to own up
+    elif persistent._mas_apology_time_db != {}:
+        m 2tfc "[player], if you have something to apologize for, please just say it."
+        m 2rfc "It means a lot more if you'd just admit what you did."
+
+    #She knows there's a reason for your apology but won't comment on it
+    else:
+        #Since this 'reason' technically varies, we don't really have a choice as we therefore can't add 0 to the db
+        #So recover a tiny bit of affection
+        $ mas_gainAffection(modifier=0.1)
+        m 2tkd "What you did wasn't funny, [player]."
+        m 2dkd "Please be more considerate about my feelings in the future."
+
+    #We only want this for actual apology reasons. Not the 0 case or the None case.
+    if mas_apology_reason:
+        #Update the apology_reason count db (if not none)
+        $ persistent._mas_apology_reason_use_db[mas_apology_reason] = persistent._mas_apology_reason_use_db.get(mas_apology_reason,0) + 1
+
+        if persistent._mas_apology_reason_use_db[mas_apology_reason] == 1:
+            #Restore a little bit of affection
+            $ mas_gainAffection(modifier=0.2)
+        elif persistent._mas_apology_reason_use_db[mas_apology_reason] == 2:
+            #Restore a little less affection
+            $ mas_gainAffection(modifier=0.1)
+
+        #Otherwise we recover no affection.
+
+    #Reset the apology reason
+    $ mas_apology_reason = None
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.apology_database,
+            eventlabel="mas_apology_bad_nickname",
+            prompt="...for calling you a bad name.",
+            unlocked=False
+        ),
+        code="APL"
+    )
+
+label mas_apology_bad_nickname:
+        if mas_getEV('mas_apology_bad_nickname').shown_count == 0:
```

Comment:
indentation here seems a bit too much.

Also, rather than always checking db for the event, just save the shown count into a local var and then make the comparisons.